src/frontend/src/components/ChangesList.tsx

```diff
@@ -3,38 +3,47 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { Link } from '@mui/material';
+import { Box, Link } from '@mui/material';
 import Typography from '@mui/material/Typography';
 import { ImplementedChange } from 'shared';
 import { fullNamePipe, datePipe } from '../utils/pipes';
 import { Link as RouterLink } from 'react-router-dom';
 import { routes } from '../utils/routes';
-import BulletList from './BulletList';
 import DynamicTooltip from './DynamicTooltip';
 
 interface ChangesListProps {
   changes: ImplementedChange[];
 }
 
+const styles = {
```

Comment:
It will be better if we give more details on this variable name. Like changeListStyles for ex.

Reply:
Id delete it and put it inline like we do everywhere else